### PR TITLE
Basic export

### PR DESCRIPTION
**Pull request by abhinav3967**
just a basic export

### CODE REVIEW

**Review by Technohacker — CHANGES_REQUESTED** [review]
Can we have the menu scene have all its own dialog boxes without going to the parent? There'd be less breakage that way if the scene was moved

**Comment by abhinav3967** [comment]
> Can we have the menu scene have all its own dialog boxes without going to the parent? There'd be less breakage that way if the scene was moved

wdym like the export dialoge be in the top menu?

**Comment by Technohacker** [comment]
> wdym like the export dialoge be in the top menu?

The code as of now contains some calls to get_parent(), if possible can you move the nodes being referenced in those calls _into_ the TopMenu scene?

File: Main.gd [hunk 1]
```diff
@@ -32,12 +36,34 @@ func load_instruments():
...
+	sequencer.connect("on_note", export_progress_bar, "set_value")
+
+func _export_finished(recorder: AudioEffectRecord, path):
+	recorder.set_recording_active(false)
+	var recording = recorder.get_recording()
+	recording.save_to_wav(path)
+	export_progress.hide()
+
 func _ready():
+	#load instrumets
```
Comment (Technohacker) [inline Main.gd:60]:
```suggestion
	#load instruments
```

**Review by Technohacker — DISMISSED** [review]

**Review by Technohacker — APPROVED** [review]

**Pull request merged**